types/artifacts.d.ts

```diff
@@ -213,6 +215,36 @@ declare global {
         requestId: string | null
       }
 
+      /** @see https://sourcemaps.info/spec.html#h.qz3o9nc69um5 */
+      export type RawSourceMap = {
+        /** File version (always the first entry in the object) and must be a positive integer. */
+        version: number
+        /** A list of original sources used by the “mappings” entry. */
+        sources: string[]
+        /** A list of symbol names used by the “mappings” entry. */
+        names: string[]
+        /** An optional source root, useful for relocating source files on a server or removing repeated values in the “sources” entry. This value is prepended to the individual entries in the “source” field. */
+        sourceRoot?: string
+        /** An optional list of source content, useful when the “source” can’t be hosted. The contents are listed in the same order as the sources in line 5. “null” may be used if some original sources should be retrieved by name. */
+        sourcesContent?: string[]
+        /** A string with the encoded mapping data. */
+        mappings: string
+        /** An optional name of the generated code that this source map is associated with. */
```

Comment:
I know this is from the spec, but no idea how to parse this sentence :) Optional filename of the source map itself? Or...yeah, what's "the generated code" here.

Reply:
Generated code = the bundle source code that was the result of this transformation / mapping.

It's a weird property because most use cases load a source map as defined in a magic comment, so the generated file is already well known. I suppose if you have a bunch of maps on disk or in memory and lost the relationship it'd be useful ...

Reply:
can you explain it in the comment :)